### ISSUE TITLE
[Research] Personal Info Revision - Research Study Topline Summary

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#34674]
## Background
Once the study is complete, we need to synthesize the data we collected and create a topline summary.

## Tasks
- [x] Create topline summary
- [x] Share w/ team
- [x] Document team feedback to incorporate into final synthesis

## Tasks
- [x] Topline summary has been created
- [x] Topline summary has been shared w/ team

**Comment by MarciMcGuireGCIO** [comment]
@cgednalske - if you have a chance, can you review this ticket and estimate or bring any questions to Sprint Planning? Thanks!

**Comment by MarciMcGuireGCIO** [comment]
@cgednalske Thank you!

**Comment by MarciMcGuireGCIO** [comment]
Updating this to include topline summary only since we had several no-shows and had to extend the study longer than originally scheduled.  Final synthesis task will be opened in a separate task for Sprint 65.  cc @Samara-Strauss .

**Comment by Samara-Strauss** [comment]
Thanks for letting me know, @MarciMcGuireGCIO, and this is totally fine! These things happen. Glad we're able to get more participants as needed.

**Comment by cgednalske** [comment]
Hi @MarciMcGuireGCIO - [Topline Summary](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/personal-information/personal-information-revision/research/topline-summary.md) has been added to our research folder! 

I will request feedback from the team in our team channel tomorrow morning once Liz and I can touch base.

**Comment by Samara-Strauss** [comment]
@cgednalske @andaleliz this is SO good. The insights are so so helpful. I'm going to get some time on the calendar to discuss these findings with the LGBTQ health services folks and Danielle. I don't think we need to discuss every finding, just the ones on which they can provide guidance. We can work together to figure out what we should present for discussion (eg. what terms we should include; how to address concerns around how this information will be used, etc) and what we don't need to discuss (eg. we probably don't need to discuss with them something like grouping together all uneditable fields). Great work!!

**Comment by MarciMcGuireGCIO** [comment]
100% agree with @Samara-Strauss - this is excellent and very informative.  @cgednalske no others apart from sharing out with the team to solicit feedback.  Thank you both for your planning and conducting a great study!

**Comment by andaleliz** [comment]
Hooray, I'm so glad to hear this feedback! @cgednalske did a great job leading the development her first topline summary. I too thought it was a really insightful study.

Christina, no other specific steps need to happen with this other than what you've outlined. I would just ask folks to reply with any feedback by end of their working day today so you can close this out and get started on the final synthesis.

Also, as a clarification, you don't need to incorporate feedback into the topliine summary - just be sure you take the feedback into consideration for the final report.

**Comment by cgednalske** [comment]
Thanks @MarciMcGuireGCIO and @andaleliz (who did a fantastic job as well!)!
And thank you for the clarification, Liz! I will have the team post any comments here and then keep those in mind moving forward with the final report.

**Comment by cgednalske** [comment]
@MarciMcGuireGCIO Looks like we do not have any additional feedback so I think we're good to close this out. If anything else happens to come up, I'll still keep it in mind for the final synthesis!

**Comment by MarciMcGuireGCIO** [comment]
Hi @cgednalske - this task didn't have an estimate on it.  Can you please add one really quick?  Thx!

**Comment by MarciMcGuireGCIO** [comment]
@cgednalske disregard - when I refreshed the page it showed up.  There's a bug that apparently has re-appeared in GitHub.
